src/system.c

```diff
@@ -1499,6 +1499,77 @@ Int8 SyNanosecondsSinceEpoch()
   return res;
 }
 
+
+/****************************************************************************
+**
+*V  SyNanosecondsSinceEpochMethod
+*V  SyNanosecondsSinceEpochMonotonic
+**  
+**  These constants give information about the method used to obtain
+**  NanosecondsSinceEpoch, and whether the values returned are guaranteed
+**  to be monotonic.
+*/
+#if defined(SYS_IS_DARWIN)
+  const char *SyNanosecondsSinceEpochMethod = "mach_absolute_time";
+  Int SyNanosecondsSinceEpochMonotonic = 1;
+#elif defined(HAVE_CLOCK_GETTIME) && defined(CLOCK_MONOTONIC)
+  const char *SyNanosecondsSinceEpochMethod = "clock_gettime";
+  Int SyNanosecondsSinceEpochMonotonic = 1;
+#elif defined(HAVE_GETTIMEOFDAY)
+  const char *SyNanosecondsSinceEpochMethod = "gettimeofday";
+  Int SyNanosecondsSinceEpochMonotonic = 0;
+#else
+  const char *SyNanosecondsSinceEpochMethod = "";
```

Comment:
How about setting `SyNanosecondsSinceEpochMethod` to e.g. `"none"` or `"unsupported"` in this case? To make it more explicit what's going on.